src/settings.js

```diff
@@ -0,0 +1,8 @@
+export const SELECT_ICON = 'selectIcon';
+
+export default new Map(Object.entries({
+    [SELECT_ICON]: {
+        symbol: 'chevron-down',
```

Comment:
Can we correct that to a sensible default that makes no assumptions about what it actually looks like? I propose `select-icon` or `select-icon-open`